### PR TITLE
Update sample google-ads.yaml file to illustrate the `use_proto_plus` flag

### PR DESCRIPTION
**Pull request by BenRKarl**

### CODE REVIEW

**Review by wihl — COMMENTED** [review]
This file is fine, but please also update the links in README.RST which are broken, notably "Version 14.0.0_"

File: google-ads.yaml [hunk 1]
```diff
@@ -7,6 +7,16 @@
 ###############################################################################
 developer_token: INSERT_DEVELOPER_TOKEN_HERE
 
+# Use proto plus
+###############################################################################
+# This parameter specifies whether the client library should return           #
+# proto-plus messages or protobuf messages. This value should be explicitly   #
+# set to either "True" or "False", For more information on the                #
+# differences between these two types, see our Protobuf Messages guide:       #
+# https://developers.google.com/google-ads/api/docs/client-libs/python/protobuf-messages #
```
Comment (laurachevalier4) [inline google-ads.yaml:16]: Nit: expand the right edge of the table so it's as wide as the link (unless that's more anti-pep-8?)

**Review by wihl — APPROVED** [review]

File: google-ads.yaml (hunk 1 above)
Comment (BenRKarl) [reply google-ads.yaml:16]: Since this isn't a Python file I don't think it is subject to pep-8, so I updated it

**Comment by BenRKarl** [comment]
@wihl I fixed the README link

**Review by laurachevalier4 — DISMISSED** [review]

**Review by wihl — APPROVED** [review]

**Pull request merged**